frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/raw-capacity-card/capacity-card.tsx

```diff
@@ -0,0 +1,189 @@
+import * as React from 'react';
```

Comment:
Moved all common parts of this card to re-use from the overall dashboard and pool dashbaord

Reply:
This is already the existing raw_capacity card code for the cluster dashboard, To re-use the same code in the pool dashboard also, I moved fetching metrics values in a separate component and passing this component to create a raw_capacity card.

pool dashboard passes its own metrics and cluster dashboard will passes its own metrics to this generic component.